test/core/iomgr/endpoint_tests.c

```diff
@@ -117,7 +117,7 @@ struct read_and_write_test_state {
 
 static void read_and_write_test_read_handler(grpc_exec_ctx *exec_ctx,
                                              void *data, grpc_error *error) {
-  struct read_and_write_test_state *state = data;
+  struct read_and_write_test_state *state = (read_and_write_test_state *)data;
```

Comment:
But this will not work in C anymore, right? There is no typedef, is there?

Reply:
Good catch! I completely forgot that the tests won't be built with the normal make

Reply:
Done! Thanks!